counterparty-lib/counterpartylib/server.py

```diff
@@ -450,6 +460,57 @@ def initialise_config(
 
     config.RPC_BATCH_SIZE = rpc_batch_size
 
+    # API V2 Settings
+
+    if api_host:
+        config.API_HOST = api_host
+    else:
+        config.API_HOST = "localhost"
+
+    if api_port:
+        config.API_PORT = rpc_port
+    else:
+        if config.TESTNET:
+            if config.TESTCOIN:
```

Comment:
Do you know what this is?

Reply:
no idea! I copy/past from RPC_*

Reply:
lol, okay I'll create an issue for removing it later :)

Reply:
ok :)